src/webviews/extension-side/serviceRegistry.web.ts

```diff
@@ -8,17 +8,34 @@ import { WebviewViewProvider } from './webviewViews/webviewViewProvider';
 import { WebviewPanelProvider } from './webviewPanels/webviewPanelProvider';
 import { IExtensionSingleActivationService } from '../../platform/activation/types';
 import { VariableViewActivationService } from './variablesView/variableViewActivationService';
-import { IVariableViewProvider } from './variablesView/types';
+import { INotebookWatcher, IVariableViewProvider } from './variablesView/types';
 import { VariableViewProvider } from './variablesView/variableViewProvider';
 import { JupyterVariableDataProvider } from './dataviewer/jupyterVariableDataProvider';
 import { JupyterVariableDataProviderFactory } from './dataviewer/jupyterVariableDataProviderFactory';
-import { IJupyterVariableDataProvider, IJupyterVariableDataProviderFactory } from './dataviewer/types';
+import {
+    IDataViewerFactory,
+    IJupyterVariableDataProvider,
+    IJupyterVariableDataProviderFactory
+} from './dataviewer/types';
 import { DataViewerCommandRegistry } from './dataviewer/dataViewerCommandRegistry';
 import { CommandRegistry as ExportCommandRegistry } from './import-export/commandRegistry';
+import { NotebookWatcher } from './variablesView/notebookWatcher';
+import { DataViewerFactory } from './dataviewer/dataViewerFactory';
+import { ExtensionSideRenderer, IExtensionSideRenderer } from './renderer';
 
 export function registerTypes(serviceManager: IServiceManager, _isDevMode: boolean) {
     serviceManager.add<IWebviewViewProvider>(IWebviewViewProvider, WebviewViewProvider);
     serviceManager.add<IWebviewPanelProvider>(IWebviewPanelProvider, WebviewPanelProvider);
+
```

Comment:
DataViewer, Varaible Views and renderer registration moved out of `src/platform`